docs/examples/framework/how-tos.md

```diff
@@ -0,0 +1,382 @@
+---
+category: examples-framework
```

Comment:
We need a new category as those examples aren't related just to the framework. TBD.

Reply:
That would fit in FAQ, but it's kinda obscure section.